client/verta/tests/unit_tests/deployment/test_deployed_model.py

```diff
@@ -380,16 +385,22 @@ def test_predict_400_error_message_missing(mocked_responses) -> None:
         token=TOKEN,
     )
     with pytest.raises(HTTPError) as err:
-        dm.predict(x=['test_prediction'])
+        dm.predict(x=["test_prediction"])
     assert str(err.value)[:-30] == (
-        '400 Client Error: Bad Request for url: '
-        'https://test.dev.verta.ai/api/v1/predict/test_path at '
+        "400 Client Error: Bad Request for url: "
+        "https://test.dev.verta.ai/api/v1/predict/test_path at "
     )
 
 
 def test_batch_predict_with_one_batch_with_no_index(mocked_responses) -> None:
-    """ Call batch_predict with a single batch. """
-    expected_df = pd.DataFrame({"A": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], "B": [11, 12, 13, 14, 15, 16, 17, 18, 19, 20]})
+    """Call batch_predict with a single batch."""
+    pd = pytest.importorskip("pandas")
```

Comment:
We have a few unit tests that _do_ need pandas to be meaningful. I'm leaving them as `importorskip` for now (which is essentially what it was [before](https://github.com/VertaAI/modeldb/pull/4103/files#diff-9688ee7a7d531768bd6b9abe27b949a8375180843a1cef576fb2d0710e28c5eaL13-L14)).